README.md

```diff
@@ -171,15 +132,26 @@ token_list = [
         "tokens": ["26009"]
     }
 ]
-sws = SmartWebSocketV2(AUTH_TOKEN, API_KEY, CLIENT_CODE, FEED_TOKEN)
+# simple retry mechanism
+sws = SmartWebSocketV2(AUTH_TOKEN, API_KEY, CLIENT_CODE, FEED_TOKEN,max_retry_attempt=2, retry_strategy="simple", retry_delay=10, retry_duration=30)
```

Comment:
retry_strategy="simple" 
it should be 0 or in other int.

Reply:
Removed the simple and exponential from string comparison from websocket and use 0 and 1